src/main/perl/RuleBasedEditor.pm

```diff
@@ -148,27 +152,27 @@ end of the line if it is modified by L<CAF::RuleBasedEditor>.
 
 =item *
 
-LINE_FORMAT_KEY_VAL:        key val (e.g. Xrootd, Apache)
+LINE_FORMAT_KW_VAL:        key val (e.g. Xrootd, Apache)
```

Comment:
Exactly this is why I try to clean it up! And I am open to any suggestion better than the current one! The change is pretty easy to do at this point, just a matter of a couple of sed commands.